music21/configure.py

```diff
@@ -1596,7 +1607,7 @@ def run(self, forceList=None):
 _DOC_ORDER = []
 
 
-class TestUserInput(unittest.TestCase):  # pragma: no cover
+class TestExternal(unittest.TestCase):  # pragma: no cover
```

Comment:
This is not named TestExternal so that it won't run on CI, where we would have to patch the interactive user inputs so that the job doesn't hang.

Reply:
I don't understand the comment.

Reply:
The class must be called TestExternal.